### PR TITLE
Cmr 4451

### PR DESCRIPTION
**Pull request by mschmele**

### CODE REVIEW

File: search-app/src/cmr/search/services/query_execution/has_granules_revised_at_feature.clj [hunk 1]
```diff
@@ -0,0 +1,24 @@
+(ns cmr.search.services.query-execution.has-granules-revised-at-feature
+  "Supports the has_granules_revised_at query parameter for collections. Returns collections
+  which have any granules that were created within the time ranges specified by the
+  has_granules_created_at time range(s).
```
Comment (chris-durbin) [inline search-app/src/cmr/search/services/query_execution/has_granules_revised_at_feature.clj]: has_granules_revised_at

File: search-app/src/cmr/search/services/query_service.clj [hunk 2]
```diff
@@ -282,24 +283,6 @@
         results (qe/execute-query context query)]
     (common-search/search-results->response context query results)))
 
-(defn get-collections-from-new-granules
```
Comment (chris-durbin) [inline search-app/src/cmr/search/services/query_service.clj]: Good catch of old unused code.

File: search-app/src/cmr/search/services/query_execution/has_granules_revised_at_feature.clj [hunk 3]
```diff
@@ -0,0 +1,24 @@
+(ns cmr.search.services.query-execution.has-granules-revised-at-feature
+  "Supports the has_granules_revised_at query parameter for collections. Returns collections
+  which have any granules that were created within the time ranges specified by the
```
Comment (ygliuvt) [inline search-app/src/cmr/search/services/query_execution/has_granules_revised_at_feature.clj]: Change "created" to "created or updated".

File: search-app/docs/api.md [hunk 4]
```diff
@@ -1206,6 +1207,14 @@ Find collections that match all of the 'project' param values
 
     curl "%CMR-ENDPOINT%/collections?has_granules_created_at\[\]=2015-01-01T10:00:00Z,"
 
+#### <a name="c-with-revised-granules"></a> Find collections with granules revised inside of a given range
```
Comment (ygliuvt) [inline search-app/docs/api.md]: change "revised" to "created or updated".

**Pull request merged**